sdk/tracing/azure-core-tracing-opencensus/README.md

```diff
@@ -0,0 +1,128 @@
+# Azure Tracing client library for Java
+This package enables distributed tracing across Azure SDK Java libraries through [Opencensus][opencensus]. OpenCensus is an open source, vendor-agnostic, single distribution of libraries to provide metrics collection and distributed tracing for services. 
+The Azure core tracing package provides:
+- Context propogation, in order to correlate activities and requests between services with an initial customer action.
+- Tracing user requests to the system, allowing to pinpoint failures and performance issues.
+
+[Source code][source_code] | [API reference documentation][api_documentation] | [Product
+documentation][aztracing_docs] | [Samples][samples]
+
+## Getting started
+### Prerequisites
+- Java Development Kit (JDK) with version 8 or above
+- [Maven][maven]
+```xml
+<dependency>
+    <groupId>io.opencensus</groupId>
+    <artifactId>opencensus-api</artifactId>
+    <version>${opencensus.version}</version>
```

Comment:
Would probably replace this and the ones below with a version number. Or define a property above with that name.

Reply:
Changed to version number